charts/azuremonitor-containers/templates/omsagent-daemonset.yaml

```diff
@@ -111,6 +111,10 @@ spec:
        - name: CUSTOM_METRICS_ENDPOINT
          value: {{ .Values.omsagent.metricsEndpoint | quote }}
        {{- end }}
+       {{- if ne .Values.omsagent.resourceEndpoint "<your_resource_endpoint>" }}
```

Comment:
Please note that, our base branch is ci_prod so you will have to make PRs against ci_prod branch not the ci_dev.

Reply:
Got it, rebased.